pandas/tests/io/test_html.py

```diff
@@ -171,7 +171,7 @@ def test_banklist_url(self):
     @tm.network
     def test_spam_url(self):
         url = (
-            "https://raw.githubusercontent.com/pandas-dev/pandas/master/"
+            "https://raw.github.com/pandas-dev/pandas/main/"
```

Comment:
Nit: Could you keep `githubusercontent` here? It redirects to that domain from raw.github.com

Reply:
When I try going to https://raw.githubusercontent.com/pandas-dev/pandas/main/, I get `400: Invalid request`.  Is that what you want?

Reply:
The string is continuing on the next line here: https://raw.githubusercontent.com/pandas-dev/pandas/main/pandas/tests/io/data/html/spam.html